common/thorhelper/thorsoapcall.cpp

```diff
@@ -1645,7 +1645,7 @@ class CWSCAsyncFor : implements IWSCAsyncFor, public CInterface, public CAsyncFo
         content = contentDecoded;
 
         if (soapTraceLevel > 6 || master->logXML)
-            multiLog(master->logctx, "Content decoded. Original %s %d", CONTENT_LENGTH, contentLength);
+            master->logctx.CTXLOG("Content decoded. Original " CONTENT_LENGTH " %d", contentLength);
```

Comment:
Is this supposed to say mCTXLOG ?

Reply:
no, deliberately changed, as can't be multi line

Reply:
Not sure about this change. I think you've decided Not to log the contents any more - but there seems to be an explicit setting master->logXML requesting that you do...

Reply:
It wasn't/isn't logging the content just a static string,CONTENT_LENGTH :
```
#define CONTENT_LENGTH "Content-Length: "
```